dts/arm/st/mem.h

```diff
@@ -91,6 +91,9 @@
 #elif defined(CONFIG_SOC_STM32F746XG)
 #define DT_FLASH_SIZE		__SIZE_K(1024)
 #define DT_SRAM_SIZE		__SIZE_K(320)
+#elif defined(CONFIG_SOC_STM32F769XI)
+#define DT_FLASH_SIZE		__SIZE_K(1024)
```

Comment:
ST site mentions 2048 flash size for STM32F769XI SoCs.

Reply:
so sorry!